draft-ietf-rtcweb-jsep.xml

```diff
@@ -2529,6 +2553,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 and its presence or absence flagged and stored.</t>
 		      <t>Any "a=imageattr" attributes MUST be parsed as specified in
                 <xref target="RFC6236"/>, Section 3, and their values stored.</t>
+              <t>If present, any "a=rid" lines MUST be parsed as specified in
+	      <xref target="I-D.ietf-mmusic-rid"></xref>, Section 10, and their values stored.</t>
+              <t>If present, any "a=simulcast" lines MUST be parsed as specified
```

Comment:
In general, yes.  But it's mostly to offer different options, not for the answer.  So we can constrain it.  I changed it to parse one and only one and ignore the rest.